src/Symfony/Component/Translation/Translator.php

```diff
@@ -132,6 +132,15 @@ public function addResource($format, $resource, $locale, $domain = null)
         }
     }
 
+    /**
+     * @param string                    $domain
+     * @param MessageFormatterInterface $formatter
```

Comment:
to be removed :)

Reply:
Thank you. I added `: void` as return type.